lib/builtin.t

```diff
@@ -51,6 +51,17 @@ package FetchStoreCounter {
     is(prototype(\&builtin::is_bool), '$', 'is_bool prototype');
 }
 
+# float contants
+{
+    use builtin qw( inf nan );
+
+    ok(inf, 'inf is true');
+    ok(inf > 1E10, 'inf is bigger than 1E10');
+    ok(inf == inf, 'inf is equal to inf');
+
+    ok(nan != nan, 'NaN is not equal to NaN');
```

Comment:
I would like to see more tests here. `>=`, `>`, `<=`, `<`, and `==`, should all return false.

Reply:
I could do a couple more tests but I didn't want to go overboard. I'm not intending to test the concept of "inf" itself, I'm sure that's handled elsewhere. I'm just checking that I've wrapped the right thing.